src/login/configSelection.tsx

```diff
@@ -201,13 +204,17 @@ function ConfigSelection({ configError, setConfigError, themeManager }: any) {
                   projectId={projectId}
                   region={region}
                   onRegionChange={region => setRegion(region)}
+                  setProjectAccess={setProjectAccess}
                 />
               </div>
               <div className="save-overlay">
                 <Button
                   variant="contained"
                   disabled={
-                    isSaving || projectId.length == 0 || region.length == 0 || gcloudRegion === region
+                    isSaving ||
+                    projectId.length === 0 ||
+                    region.length === 0 ||
+                    (gcloudRegion === region && gcloudProject === projectId)|| !projectAccess
```

Comment:
This doesn't work the way you think it would, because you never update gcloud project and gcloud region on successful save, so changing projects and changing it back would fail.

Also it doesn't address the issue the project and region being changed outside of jupyter.

Reply:
Lets just revert the configSelection changes and put in into a separate pull request if possible?

Reply:
reverted config changes